en/asgardeo/docs/apis/restapis/actions.yaml

```diff
@@ -0,0 +1,859 @@
+          enum:
+            - PRE_ISSUE_ACCESS_TOKEN
+            - PRE_UPDATE_PASSWORD
+            - PRE_UPDATE_PROFILE
+            - PRE_REGISTRATION
+        name:
+          type: string
+          example: Access Token Pre Issue
+          minLength: 1
+          maxLength: 255
+        description:
+          type: string
+          example: This is the configuration of pre-action for issuing access token.
+          maxLength: 255
+        status:
+          type: string
+          enum:
+            - ACTIVE
+            - INACTIVE
+        endpoint:
+          $ref: '#/components/schemas/EndpointResponse'
+
+    ActionBasicResponse:
+      type: object
+      properties:
+        id:
+          type: string
+          example: '24f64d17-9824-4e28-8413-de45728d8e84'
+        type:
+          type: string
+          enum:
+            - PRE_ISSUE_ACCESS_TOKEN
+            - PRE_UPDATE_PASSWORD
+            - PRE_UPDATE_PROFILE
+            - PRE_REGISTRATION
+        name:
+          type: string
+          example: Access Token Pre Issue
+          minLength: 1
+          maxLength: 255
+        description:
+          type: string
+          example: This is the configuration of pre-action for issuing access token.
+          maxLength: 255
+        status:
+          type: string
+          enum:
+            - ACTIVE
+            - INACTIVE
+
+    ActionResponseList:
+      type: array
+      items:
+        $ref: '#/components/schemas/ActionResponse'
+
+    ActionTypesResponse:
+      type: array
+      items:
+        $ref: '#/components/schemas/ActionTypesResponseItem'
+
+    ActionTypesResponseItem:
+      type: object
+      properties:
+        type:
+          type: string
+          enum:
+            - PRE_ISSUE_ACCESS_TOKEN
+            - PRE_UPDATE_PASSWORD
+            - PRE_UPDATE_PROFILE
+            - PRE_REGISTRATION
+        displayName:
+          type: string
+          example: "Pre Issue Access Token."
+        description:
+          type: string
+          example: "Extension point configuration for Pre Issue Access Token."
+        count:
+          type: integer
+          example: 2
+        self:
+          type: string
+          example: "/t/{organization-name}/api/server/v1/actions/preIssueAccessToken"
+
+    AuthenticationTypeResponse:
+      type: object
+      required:
+        - type
+      properties:
+        type:
+          type: string
+          enum:
+            - NONE
+            - BEARER
+            - API_KEY
+            - BASIC
+          example: BASIC
+
+    ActionUpdateModel:
+      type: object
+      properties:
+        name:
+          type: string
+          example: Access Token Pre Issue
+          minLength: 1
+          maxLength: 255
+        description:
+          type: string
+          example: This is the configuration of pre-action for issuing access token.
+          maxLength: 255
+        endpoint:
+          $ref: '#/components/schemas/EndpointUpdateModel'
```

Comment:
Ideally these properties should have a description

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4698/commits/ee4f5ca7c24d3d8dcaca4943b7b5aadb6e16e7fc